iothub/device/tests/ClientPropertyCollectionTests.cs

```diff
@@ -160,24 +160,46 @@ public void ClientPropertyCollection_CanAddMultiplePropertyAndGetBackWithoutDevi
             outIntValue.Should().Be(IntPropertyValue);
         }
 
+        [TestMethod]
+        public void ClientPropertyCollection_TryGetValueShouldReturnFalseIfValueNotFound()
+        {
+            var clientProperties = new ClientPropertyCollection();
```

Comment:
It is preferred to add //arrange, //act, //assert as comments appropriately in test code.